### ISSUE TITLE
different icons for 2 different targets -- wtf

### ISSUE THREAD

**Issue by horatio-sans-serif** [AlexDenisov/FontasticIcons#2]
Hi, I have a weird issue. I have 2 targets for a client (free, pro -- yes, antiquated but whatever).

I am using FontasticIcons in both targets via CocoaPods.

It works great for the one target but I'm seeing a different icon in the other target! 

I have no idea why and haven't had time to dive into your code yet. I am wondering if you knew off the top of your head what's up.

Here are the images

http://imgur.com/moIN4,k5kkt#0

http://imgur.com/moIN4,k5kkt#1

I am using 0.1.0 BTW

**Comment by horatio-sans-serif** [comment]
Figured it out... CocoaPods didn't update the XCode project's supplemental target with a script build phase that copied resources for dependent pods. Thus, the icon fonts weren't being copied and thus the code points used glyphs from the backup font which I suppose is something like Helvetica.

I added this to the target in which it was missing. I hope this helps someone someday.

/bin/sh
"${SRCROOT}/Pods/Pods-resources.sh"
